src/_data/user_guide_navigation.yml

```diff
@@ -67,6 +67,8 @@ toc:
         url: /rest.li/Send-Rest_li-Request-Query-In-Request-Body
       - page: Use Projections
         url: /rest.li/How-to-use-projections-in-Java
+      - page: Use PathSpec in Rest.Li
```

Comment:
I don't think this is appropriate for the "How to" subheading. This is less of a guide and more of a specification for the PathSpec concept. Tbh I think this would be more appropriate in most of the other sections (eg. Data Schemas, Data, Main Concepts, etc.)

Reply:
I thought about putting this file there. But I had two reasons
(1) most of the contents in this files are just examples, looks more fitting to "how to" section.
(2) This concept is not very big and has been mentioned in `projection` or `validation`. I don't think there needs to be a separate page for that. In fact if we have had too many files in those two secionts `Data Schema` and `Main Concept`, it adds burden to the users to understand.

I mainly want to keep this is a reference guide.

Reply:
I agree with Evan on this. 
Normally "How to" pages should answer how to solve a specific problem. This can go under the "Main Concepts" section?